source/Graphite.System/AppPoolListener.cs

```diff
@@ -11,65 +11,84 @@ namespace Graphite.System
     internal class AppPoolListener
     {
         private readonly string appPoolName;
+        private readonly string category = "Process";
+        private readonly string counter = "Working Set";
 
         private string counterName;
 
-        private CounterListener workingSetListener;
+        private CounterListener counterListener;
+        private bool workingSet;
```

Comment:
I think for simplicity the decision whether it is a "classic" working set counter or a new one should be made outside of this class, leaving only the new constructor which requires the counter parameter.